packages/components/form/__specs__/__snapshots__/text_field.spec.tsx.snap

```diff
@@ -0,0 +1,225 @@
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-flex-direction: row;
+  -ms-flex-direction: row;
+  flex-direction: row;
+}
+
+.c3 {
+  font-family: -apple-system,BlinkMacSystemFont,San Francisco,Segoe UI, Roboto,Helvetica Neue,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 600;
+}
+
+.c11 {
+  font-family: -apple-system,BlinkMacSystemFont,San Francisco,Segoe UI, Roboto,Helvetica Neue,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-style: none;
+  font-size: 0.75rem;
+  font-weight: 400;
+  line-height: 1rem;
+}
+
+.c9 {
+  outline: none;
+  background: none;
+  border: none;
+  color: inherit;
+  fontsize: inherit;
+  line-height: inherit;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  text-align: inherit;
+  box-sizing: border-box;
+}
+
+.c5 {
+  font-family: -apple-system,BlinkMacSystemFont,San Francisco,Segoe UI, Roboto,Helvetica Neue,sans-serif;
+  display: inline-block;
+  background-color: #FFFFFF;
+  border: 1px solid;
+  border-radius: 4px;
+  color: #101112;
+  font-size: 0.875rem;
+  height: 40px;
+  line-height: 1rem;
+  margin: 0;
+  padding: 0.75rem 0.75rem;
+  width: 300px;
+}
+
+.c5 transition:"background 0.75s ease",
+.c5 input::-webkit-input-placeholder {
+  opacity: 1;
+  color: #9CA1A5;
+  -webkit-text-fill-color: $({ theme}) => theme.colors.text.tertiary;
+}
+
+.c5 transition:"background 0.75s ease",
+.c5 input::-moz-placeholder {
+  opacity: 1;
+  color: #9CA1A5;
+  -webkit-text-fill-color: $({ theme}) => theme.colors.text.tertiary;
+}
+
+.c5 transition:"background 0.75s ease",
+.c5 input:-ms-input-placeholder {
+  opacity: 1;
+  color: #9CA1A5;
+  -webkit-text-fill-color: $({ theme}) => theme.colors.text.tertiary;
+}
+
+.c5 transition:"background 0.75s ease",
+.c5 input::placeholder {
+  opacity: 1;
+  color: #9CA1A5;
+  -webkit-text-fill-color: $({ theme}) => theme.colors.text.tertiary;
+}
+
+.c5:hover {
+  border-color: #46494D;
+}
+
+<form
+    autocomplete="off"
+    class="c0"
+    data-testid="test-form"
+    id="test-form"
+  >
+    <div
+      class="c1 "
+      data-testid="form-field-my-text-field"
+      display="flex"
+      width="input"
+    >
+      <label
+        class="c2 c3"
+        color="text.primary"
+        font-weight="600"
+        for="my-text-field"
+      >
+        This is my text field
+      </label>
+      <div
+        class="c4"
+      >
+        <div
+          class="c0 c5"
+          data-testid="input-wrapper-text-field-test"
+        >
+          <div
+            class="c6 c7"
+            display="flex"
+          >
+            <div
+              class="c8"
+            >
+              <input
+                aria-invalid="false"
+                class="c9"
+                data-testid="input-text-field-test"
+                id="my-text-field"
+                name="myTextField"
+                placeholder=""
```

Comment:
The placeholder is blank which leads me to believe our translations for the placeholder are not right in our text field spec.

Reply:
Right! Translations for placeholder fixed in b848025